pkg/logs/logs.go

```diff
@@ -27,19 +27,22 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/apimachinery/pkg/util/wait"
-	"k8s.io/klog"
-	"k8s.io/klog/klogr"
+	"k8s.io/klog/v2"
+	"k8s.io/klog/v2/klogr"
 
 	"github.com/jetstack/cert-manager/pkg/api"
 )
 
 var (
 	Log = klogr.New().WithName("cert-manager")
 
-	ErrorLevel = 0
-	WarnLevel  = 1
-	InfoLevel  = 2
-	DebugLevel = 3
+	// following analog to https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md
+	ErrorLevel        = 0
+	WarnLevel         = 1
+	InfoLevel         = 2
+	ExtendedInfoLevel = 3
+	DebugLevel        = 4
+	TraceLevel        = 5
```

Comment:
Could  / should we make these constants and give them a type?

Reply:
made then consts not sure why they were not before...
typing then will break the interface